galpy/potential/TwoPowerSphericalPotential.py

```diff
@@ -129,15 +161,13 @@ def _Rforce(self,R,z,phi=0.,t=0.,_forceFloatEval=False):
         HISTORY:
            2010-07-09 - Written - Bovy (NYU)
         """
-        if not _forceFloatEval and not self.integerSelf == None:
+        if not _forceFloatEval and self.integerSelf is not None:
             return self.integerSelf._Rforce(R,z,phi=phi,t=t)
-        else:
-            r= numpy.sqrt(R**2.+z**2.)
-            return -R/r**self.alpha*self.a**(self.alpha-3.)/(3.-self.alpha)\
-                *special.hyp2f1(3.-self.alpha,
-                                self.beta-self.alpha,
-                                4.-self.alpha,
-                                -r/self.a)
+        # else:
+        a, alpha, beta = self.a, self._alpha, self._beta
+        r= sqrt(R**2.+z**2.)
+        return -R/r**alpha*a**(alpha-3.)/(3.-alpha)\
+            *hyp2f1(3.-alpha, beta-alpha, 4.-alpha, -r/a)
```

Comment:
Why did you re-write this, nothing seems changed? It's good practice to show clearly that this is an ``else`` statement (and it's bad practice of mine to return in the middle of the function...).